docs/content/docs/how-does-opa-work.md

```diff
@@ -229,6 +229,18 @@ allow {
 }
 ```
 
+To supply the input document, query OPA using the POST method.
+
+POST /v1/data/opa/examples/allow HTTP/1.1
+Content-Type: application/json
+{
+  "input": {
+    "method": "GET",
+    "path": "/servers/s2",
+    "user": "alice"
+  }
+}
```

Comment:
Please surround the block with ``` like the other ones in the file so that markdown will format this correctly. 

Check out the current results: https://deploy-preview-1344--openpolicyagent.netlify.com/docs/how-does-opa-work/  (link is in the "netlify/openpolicyagent/deploy-preview " check details)

Reply:
woops! good call! Fixed.